src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export class Actions {
```

Comment:
Hmmm I see. Do you think there is a strong advantage to using an enum? I do see the advantage to compare values as numbers instead of strings, but I am having trouble implementing it with the ion-segment . The ion-segment component works with strings, so the comparions do not work with the enum. I'd have to convert everything between strings and numbers, which feels wrong. Maybe I'm missing something else though?